scripts/v.import/v.import.py

```diff
@@ -121,6 +123,48 @@ def cleanup():
         grass.try_remove(SRCGISRC)
 
 
+def checkGDALversion(smallerversion="2.4.1"):
+    """Checks GDAL version if it is smaller than given smaller version
+
+    :param smallerversion: version number to check if the actual GDAL version
+                           is smaller
+    :type smallerversion: str
+    """
+    smallerversion_tuple = tuple([int(x) for x in smallerversion.split('.')])
+    GDALversion = grass.parse_command('g.version', flags='reg')['gdal']
+    GDALversion_tuple = tuple([int(x) for x in GDALversion.split('.')])
```

Comment:
Given the problems with parsing versions (see recent issues with wxPython and CPython version numbers), please double check what version format GRASS guarantees here and make the parsing more robust if needed.

Reply:
GRASS gives the version which stands in the gdal_version.h GDAL_RELEASE_NAME link this one: https://github.com/rwinlib/gdal2/blob/master/include/gdal/gdal_version.h#L29.
It is robust enough?

Reply:
In GRASS GIS mostly `GDAL_VERSION_NUM` is used (see for example
https://github.com/OSGeo/grass/blob/5b4722aee674275f91e5e08d930df18b0c93c530/vector/v.in.ogr/main.c#L430
).

However, in g.version also `GDAL_RELEASE_NAME` is used:
https://github.com/OSGeo/grass/blob/5b4722aee674275f91e5e08d930df18b0c93c530/general/g.version/main.c#L215

Both origin from /usr/include/gdal/gdal_version.h.

To me both variants look fine.

@rouault: is the usage of `GDAL_RELEASE_NAME` which is easy to parse in Python discouraged to be used?

Reply:
`GDAL_RELEASE_NAME` can be something like "3.1.0dev", in which case the "dev" part would need to be removed, otherwise integer conversion and comparison will not work.

Reply:
ttps://github.com/qgis/QGIS/blob/master/tests/src/python/test_provider_ogr_gpkg.py#L49 + https://github.com/qgis/QGIS/blob/master/tests/src/python/test_provider_ogr_gpkg.py#L242 can be used for inspiration

Reply:
Thanks a lot. The example is very helpful